user.go

```diff
@@ -89,3 +89,28 @@ func (a *App) DeactivateUser(userID string) (*User, error) {
 
 	return &user, nil
 }
+
+func (a *App) UpdateUserEmail(userID string, email string) (*User, error) {
+	type respUser struct {
+		User User `json:"user"`
+	}
+	var userBody respUser
+
+	response, err := resty.New().R().
+		SetAuthToken(a.Config.APIKey).
+		SetResult(&userBody).
+		SetBody(fmt.Sprintf(`{"email": %v}`, email)).
```

Comment:
I believe the correct way to do this is using a quick struct

type updateBody struct {
      Email string `json:"email"`
}
...
then use that struct in the SetBody